ms-ml/app/routers/predict.py

```
@@ -0,0 +1,163 @@
+        if r2 > 0.7:
+            confidence = "high"
+        elif r2 > 0.4:
+            confidence = "medium"
+        else:
+            confidence = "low"
+
+        return WaitTimeResponse(
+            predicted_wait_days=round(wait_days, 2),
+            confidence=confidence,
+        )
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)